query.go

```diff
@@ -1424,7 +1424,7 @@ func (q *Query) Buffer(attributeOrDimension string) (interface{}, error) {
 		length := (*cbufferSize) / C.sizeof_int64_t
 		buffer = (*[1 << 46]int64)(cbuffer)[:length:length]
 
-	case TILEDB_UINT8:
+	case TILEDB_UINT8, TILEDB_BLOB:
```

Comment:
Is this an intentional related change?

Reply:
It's fine that it's here, just wanted to make sure there wasn't other stuff that was missing for it to work.